### PR TITLE
Stop using deprecated post-release versioning scheme

### PR DESCRIPTION
**Pull request by nforro**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,4 +7,4 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
-version_scheme = "post-release"
+version_scheme = "no-guess-dev"
```
Comment (lachmanfrantisek) [inline pyproject.toml:10]: Would it be possible to mention why we are using this schema? I always need to google it when I see it..;)

Comment (nforro) [reply pyproject.toml]: @lachmanfrantisek does this work? :slightly_smiling_face: 

https://github.com/packit/specfile/blob/446a315ca546b924627e4c89e19a4f654ff8609a/pyproject.toml#L10-L16

Comment (lachmanfrantisek) [reply pyproject.toml]: Sure, thanks a lot!

**Review by lachmanfrantisek — APPROVED** [review]

**Pull request merged**